DwarfCorp/DwarfCorpXNA/GameStates/PlayState.cs

```diff
@@ -225,7 +225,7 @@ public override void OnEnter()
                 };
 
                 World.UnpauseThreads();
-                AutoSaveTimer = new Timer(GameSettings.Default.AutoSaveTimeMinutes * 60.0f, false, Timer.TimerMode.Real);
+                AutoSaveTimer = new Timer(GameSettings.Default.AutoSaveTimeMinutes * 60.0f, false, Timer.TimerMode.Game);
```

Comment:
I'm not sure I agree with this change. While the game is paused, the user can create/remove commands and do a bunch of other stuff that won't get autosaved now.